src/pages/Admin/WorldUsers/WorldUsers.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useMemo } from "react";
+import { useForm } from "react-hook-form";
+import { Button } from "components/admin/Button";
+import { Header } from "components/admin/Header";
+import { Input } from "components/admin/Input";
+import { InviteAdminModal } from "components/admin/InviteAdminModal";
+import { WorldUserCard } from "components/admin/WorldUserCard";
+import { AdminLayout } from "components/layouts/AdminLayout";
+import { includes } from "lodash";
+
+import { UserId, WorldId } from "types/id";
+
+import { useWorldAndSpaceByParams } from "hooks/spaces/useWorldAndSpaceByParams";
+import { useWorldSpaces } from "hooks/spaces/useWorldSpaces";
+import { useProfileByIds } from "hooks/user/useProfileByIds";
+import { useShowHide } from "hooks/useShowHide";
+import { useUser } from "hooks/useUser";
+
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import { AdminRestricted } from "components/atoms/AdminRestricted";
+
+import * as TW from "./WorldUsers.tailwind";
+
+export interface WorldScheduleFormInput {
```

Comment:
Not sure if this one is used outside, but if it not, same removal for it and all the rest so we can have shorter output of `npm run prune` (different from `npm prune`)

```suggestion
interface WorldScheduleFormInput {
```